SchedulerJobs/SchedulerJobs.Common/Configuration/FeatureToggles.cs

```diff
@@ -2,30 +2,21 @@
 using System.Diagnostics.CodeAnalysis;
 using LaunchDarkly.Sdk;
 using LaunchDarkly.Sdk.Server;
-using LaunchDarkly.Sdk.Server.Interfaces;
 
 namespace SchedulerJobs.Common.Configuration
 {
     public interface IFeatureToggles
     {
         public bool StorePeopleIngestion();
-        public bool WorkAllocationToggle();
-        public bool ImportAllJudiciaryUsersToggle();
```

Comment:
we shouldn't remove the ImportAllJudiciaryUsers toggle, we still need to be able to turn it on and off

Reply:
Sorry I got confused with a different toggle. I have reinstate it